src/entities/pool.ts

```diff
@@ -229,6 +237,20 @@ export default class Pool {
 		})
 		this.committer = poolCommitter;
 
+		if (poolInfo.oracleType === KnownOracleType.SMAOracle) {
```

Comment:
I think we should flip this so it defaults to SMA. Mainly because the expected state calcs default to simulating SMA during the value transfers simulations

Reply:
mmmm, at that stage I think we should just drop the other type of ChainlinkOracleWrapper since an SMAOracle is a ChainlinkOracleWrapper

Reply:
made smaOracle default